homeassistant/components/music_assistant/manifest.json

```diff
@@ -8,6 +8,6 @@
   "iot_class": "local_push",
   "issue_tracker": "https://github.com/music-assistant/hass-music-assistant/issues",
```

Comment:
We should set that to the HA issue tracker

Reply:
it should actually be omitted - that is only for custom integrations. I'll remove it.

Reply:
Oh that's true